src/runtimeMain/kotlin/com/epam/drill/test/agent/TestListener.kt

```diff
@@ -53,6 +59,17 @@ actual object TestListener {
         ThreadStorage.stopSession()
     }
 
+    private fun isNotFinalizeTestState(test: String?): Boolean =
```

Comment:
Why did you extract this into a separate method?

Reply:
for better readability 🤷‍♂